### PR TITLE
Changelog and version bump

### PR DESCRIPTION
**Pull request by KimberlyMunoz**
Just updating the version from #36  to tag.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,8 +6,11 @@ We follow the [Semantic Versioning 2.0.0](http://semver.org/) format.
 
 ## 0.11.0 - 2015-05-13
 
+### Added
+- Added webfont variables to make it easy to override defaults in cf-theme-overrides.less
+
 ### Removed
-- Removing CFPB fonts. To add your own custom fonts,
+- Removing CFPB fonts. To add your own custom fonts, modify the `@webfont-regular/italic/medium/demi` mixins.
```
Comment (Scotchester) [inline CHANGELOG.md]: "mixins" should be "variables"

And we should probably explicitly note that unless they're choosing a system font (which is implied by calling it @webfont, anyhow), they're going to need to add their `@font-face` rules somewhere, too.

Comment (KimberlyMunoz) [inline CHANGELOG.md]: Agreed. I'll update.

**Comment by Scotchester** [comment]
:warning: **Warning to any CFPB projects using cf-core:**

You will now need to _manually include_ `licensed-fonts.css` in your project, since it will no longer come along for the ride with cf-core.

/cc @cfpb/front-end-team-admin @cfpb/front-end-team-write

**Comment by Scotchester** [comment]
Following up that last comment: You can grab the file from generator-cf and follow the additions to `cf-theme-overrides.less` shown in this PR: cfpb/generator-cf#69

File: CHANGELOG.md (hunk 1 above)
Comment (KimberlyMunoz) [inline CHANGELOG.md]: Updated!

**Comment by Scotchester** [comment]
Thanks! :+1:

(Note on failed build: repo has been activated in Travis but does not have a `travis.yml` file yet.)

**Pull request merged**